### ISSUE TITLE
Doesn't work with @mapbox/mapbox-gl-draw v1.4.1

### ISSUE THREAD

**Issue by tedi4t** [drykovanov/mapbox-gl-draw-rotate-scale-rect-mode#39]
When `@mapbox/mapbox-gl-draw v1.4.1` builds into the node_modules, there is no `src` folder (Everything is into the `dist` folder). At the same time, there are `import`s [in your code](https://github.com/drykovanov/mapbox-gl-draw-rotate-scale-rect-mode/blob/master/src/index.js#L3) from the `src` folder, so it throws an error if we try to set `@mapbox/mapbox-gl-draw` version greater than `1.3.0` and install packages.
That's because of `@mapbox/mapbox-gl-draw` updates. In older version, there was a `src` folder but now they have removed it.

**Comment by mfnash** [comment]
any update of this ?

**Comment by ghost** [comment]
Has anyone managed to make it work with mapbox-gl-draw v1.4.1 or v1.4.2?

**Comment by joseph-sm** [comment]
Yeah that got chnaged a while ago you can still access these via the `MapboxDraw` import.

```javascaript
import MapboxDraw from '@mapbox/mapbox-gl-draw';

console.log(MapboxDraw.lib);
```

<img width="755" alt="image" src="https://github.com/drykovanov/mapbox-gl-draw-rotate-scale-rect-mode/assets/54251292/73547469-e3f7-4356-893e-ac2f4aba00b5">

eg.

The following code would replace the affected imports.

```javascript
import MapboxDraw from '@mapbox/mapbox-gl-draw';
const Constants = MapboxDraw.constants;
const {
  doubleClickZoom,
  createSupplementaryPoints,
  CommonSelectors,
  moveFeatures,
} = MapboxDraw.lib;

...
```
